webviz_subsurface/containers/_parameter_distribution.py

```diff
@@ -319,7 +320,7 @@ def render_scatter(ens1, x_col, ens2, y_col, color, density):
 @cache.memoize(timeout=cache.TIMEOUT)
 def render_matrix(ensemble_path):
 
-    data = get_parameters(ensemble_path)
+    data = get_parameters(ensemble_path).select_dtypes([np.number])
```

Comment:
Are we sure `get_parameters` will always return `np.number` as numerical data type for numerical values?

Is it possible + more robust to e.g. first use [`to_numeric`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.to_numeric.html) with argument `errors='coerce'` followed by [`dropna`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.dropna.html)? That would also make `import numpy as np` unnecessary.

Reply:
Yes, seems possible and more robust :)
Fixed